pynamodb_attributes/uuid.pyi

```diff
@@ -1,6 +1,9 @@
+from typing import Any
 from uuid import UUID
 
 from ._typing import Attribute
 
+
 class UUIDAttribute(Attribute[UUID]):
-    ...
+    def __init__(self, remove_dashes: bool = False, **kwargs: Any) -> None:
```

Comment:
```suggestion
    def __init__(self, remove_dashes: bool = ..., **kwargs: Any) -> None:
```

Reply:
I thought the stub should match the function definition?